### PR TITLE
fix: Fix codecov post job

### PR DESCRIPTION
**Pull request by mateusoliveira43**
Fix silent error happening in CI. Example
```
[2023-12-11T19:59:53.180Z] ['error'] There was an error running the uploader: 
Unable to detect SHA and slug, please specify them manually.
See the help for more details.
```
Reference https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/branch-ci-openshift-oadp-operator-master-unit-test-post/1734300664726556672/artifacts/unit-test-post/unit/build-log.txt

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -274,14 +274,7 @@ endif
...
-	./codecov > tmp.results 2> tmp.err
-	cat tmp.results || echo "tmp.results not found"
-	cat tmp.err || echo "tmp.err not found"
-	@echo $$(cat tmp.results | grep 'resultURL' -c)
-	@echo $$(cat tmp.err | grep 'please specify sha and slug manually' -c)
-	if [ $$(cat tmp.err | grep 'please specify sha and slug manually' -c) == 1 ]; then \
-		echo "specifying sha and slug manually" && ./codecov -C $(shell git rev-parse HEAD) -r openshift/oadp-operator; \
-	fi
+	./codecov -C $(shell git rev-parse HEAD) -r openshift/oadp-operator
 	rm -f codecov tmp.*
```
Comment (mrnold) [inline Makefile:278]: Maybe also take out the "tmp.*" on the next line? Since they were generated in the removed lines.

Comment (mateusoliveira43) [reply Makefile]: good catch, done :smile:

**Review by kaovilai — APPROVED** [review]

**Comment by kaovilai** [comment]
/override ci/prow/4.12-e2e-test-aws
/override ci/prow/4.12-e2e-test-azure
/override ci/prow/4.13-e2e-test-azure

No e2e, or operator change.

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**